infrahub_sdk/protocols.py

```diff
@@ -397,7 +399,6 @@ class CoreNumberPool(CoreResourcePool, LineageSource):
 
 
 class CoreObjectPermission(CoreBasePermission):
-    branch: String
```

Comment:
@gmazoyer is this change expected ? it wasn't part of my change

Reply:
That should be fine as we decided to remove the "branch" parts as part of the object permissions with the argument that users probably wouldn't want to setup permissions on specific branches and instead have settings for "allow on default branch" or "allow on other branches" on in the future "allow on branches that I have created".